src/almanac.js

```diff
@@ -179,7 +178,7 @@ export default class Almanac {
    * Interprets value as either a primitive, or if a fact, retrieves the fact value
    */
   getValue (value) {
-    if (isObjectLike(value) && Object.prototype.hasOwnProperty.call(value, 'fact')) { // value = { fact: 'xyz' }
+    if (Boolean(value instanceof Object) && Object.prototype.hasOwnProperty.call(value, 'fact')) { // value = { fact: 'xyz' }
```

Comment:
We shouldn't need to wrap this in Boolean given this method is called on every value on every comparison let's try to do really minimal work here.

Reply:
shall we return to typeof?

Reply:
done :)